lib/ua_parser/storage.ex

```diff
@@ -1,46 +1,16 @@
 defmodule UAParser.Storage do
   @moduledoc """
-  Storage of User-Agent regular expressions.
+  Compile time loads regex data. Unfortunatley would require a restart for any pattern update
   """
 
-  use GenServer
-
-  alias __MODULE__, as: Storage
   alias UAParser.Processor
+  {:ok,doc} = (:code.priv_dir(:ua_parser) ++ '/patterns.yml')
```

Comment:
Could you add a space after the comma in your tuple and align the pipes?  There's also a few extra lines of whitespace in the module, please remove those.  

Additionally we could clean up the first part of the pipe with a private method:

```elixir
alias UAParser.Processor

{:ok, doc} = 
  patterns_path
  |> to_string
  |> Yomel.decode_file

data = Processor.process(doc)
@data data

def list, do: @data

defp patterns_path, do: :code.priv_dir(:ua_parser) ++ '/patterns.yml'
```